org.jacoco.core/src/org/jacoco/core/analysis/Analyzer.java

```diff
@@ -106,8 +107,16 @@ public void visitEnd() {
 	 *            reader with class definitions
 	 */
 	public void analyzeClass(final ClassReader reader) {
-		final ClassVisitor visitor = createAnalyzingVisitor(
-				CRC64.classId(reader.b), reader.getClassName());
+		analyzeClass(reader.b);
+	}
+
+	private void analyzeClass(final byte[] source) {
```

Comment:
Move this private method below the public analyze* methods?

Reply:
@marchof same as https://github.com/jacoco/jacoco/pull/629#discussion_r175766878